src/dmd/dimport.d

```diff
@@ -11,6 +11,8 @@
 
 module dmd.dimport;
 
+import core.stdc.stdio;
```

Comment:
unrelated change

Reply:
This is fine and commonly accepted to be added in PRs.

Reply:
Done.